channels/db.py

```diff
@@ -1,11 +1,12 @@
+from django.db import connections
```

Comment:
This goes with the import on the next line (which would fix the isort failure...)